environ/project.clj

```diff
@@ -1,7 +1,18 @@
-(defproject environ "1.1.0"
+(defproject environ "1.1.0-SNAPSHOT"
   :description "Library for accessing environment variables"
   :url "https://github.com/weavejester/environ"
   :scm {:dir ".."}
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.5.1"]])
+  :dependencies [[org.clojure/clojure "1.8.0"]
+                 [org.clojure/clojurescript "1.9.229"]]
```

Comment:
ClojureScript should be a provided dependency, otherwise we end up shipping ClojureScript even if people just want to use Clojure.